src/System.IO.Compression/tests/ZipArchive/zip_InvalidParametersAndStrangeFiles.cs

```diff
@@ -306,6 +306,33 @@ public static async Task Zip64ArchiveEntry_CorruptedFile_Read_UpToUncompressedSi
             }
         }
 
+
+        [Fact]
+        public static async Task UnseekableVeryLargeArchive_DataDescriptor_Read_Zip64()
+        {
+            MemoryStream stream = await LocalMemoryStream.readAppFileAsync(strange("veryLarge.zip"));
+
+            using (
+                ZipArchive archive = new ZipArchive(stream, ZipArchiveMode.Read))
+            {
+                ZipArchiveEntry e = archive.GetEntry("bigFile.bin");
+                using (var ms = new MemoryStream())
```

Comment:
Looks like this MemoryStream isn’t part of the test scenario. You can just read into buffer and ignore the read data.